### PR TITLE
Add new variable to equity data

### PR DESCRIPTION
**Pull request by maggiesu0725**

### CODE REVIEW

**Comment by maggiesu0725** [comment]
@rpitingolo @lhendey  ready for review

File: Prog/Equity_Compile_Feature_Data.sas [hunk 1]
```diff
@@ -101,80 +196,43 @@ numerator = earningover75k_&_years.;
...
-year = "2017";
-pctafford = AMI_first_afford/total_sales;
-geo= &geo
+data commute;
+length indicator $80;
+set ACS.Acs_2012_16_dc_sum_tr_&geosuf;
+keep indicator year &geo numerator denom equityvariable;
+indicator = "Workers with Travel time to work less than 45 minutes";
+year = "2012-2016";
+commuteunder40 = numownercostburden_&_years./rentcostburdendenom_&_years.;
```
Comment (lhendey) [inline Prog/Equity_Compile_Feature_Data.sas]: cost burden vars in the commuteunder40?

File: Prog/Equity_Compile_Feature_Data.sas [hunk 2]
```diff
@@ -197,17 +255,22 @@ denom = births_w_prenat_2016;
...
+data afford;
+keep indicator year &geo numerator denom equityvariable;
+set afford_&geosuf;
+length indicator $80;
+run;
+
 data equity_tabs_&geosuf;
 retain indicator year &geo numerator denom equityvariable;
-set  unemployment postsecondary homeownership income75k abovepoverty earning75k violentcrime prenatal;
+set  unemployment postsecondary homeownership income75k abovepoverty costburden commute earning75k violentcrime prenatal afford;
```
Comment (lhendey) [inline Prog/Equity_Compile_Feature_Data.sas]: child poverty?

File: Prog/Equity_Compile_Feature_Data.sas [hunk 3]
```diff
@@ -101,80 +196,43 @@ numerator = earningover75k_&_years.;
...
+data commute;
+length indicator $80;
+set ACS.Acs_2012_16_dc_sum_tr_&geosuf;
+keep indicator year &geo numerator denom equityvariable;
+indicator = "Workers with Travel time to work less than 45 minutes";
+year = "2012-2016";
+commuteunder40 = numownercostburden_&_years./rentcostburdendenom_&_years.;
+equityvariable = commuteunder40;
+denom = popemployedworkers_&_years.;
+numerator = (popemployedtravel_lt5_&_years. + popemployedtravel_10_14_&_years.+ popemployedtravel_15_19_&_years.+ popemployedtravel_20_24_&_years. + popemployedtravel_25_29_&_years. + popemployedtravel_30_34_&_years. + popemployedtravel_35_39_&_years.);
```
Comment (lhendey) [inline Prog/Equity_Compile_Feature_Data.sas]: missing 40-45?

**Comment by lhendey** [comment]
@maggiesu0725 - . i fixed the child poverty indicator (was calculated for kids below poverty). okay to leave off those sales for this purpose. actually - can you suppress all clusters with fewer than 200 people from the output?

**Comment by maggiesu0725** [comment]
@lhendey could you check my code for hiding numbers for cluster 42 and 45 towards the end? It's running but not doing what I want?

**Comment by lhendey** [comment]
@maggiesu0725 - this can be merged. Can you send the updated data to Alice? Please tell her that there are clusters that we suppressed data - i'm not sure how they want to handle that in the feature. 3 where we suppressed all indicators and a handful that may have one or two suppressed.

**Pull request merged**